### DISCUSSION TITLE
Ish iPhone phishing

### DISCUSSION THREAD

**Discussion by Itsh98** [ish-app/ish#2255] [General]
Please I know nothing about ish or phishing or any of this, can someone please explain to me step by step how to make a Snapchat phishing page

**Comment by itskenny0** [comment]
This is spam.

**Reply by Itsh98** [reply]
No it ain't it's a genuine question

**Reply by Feror-BotMaker** [reply]
A genuine question about spam.

**Reply by DuttyWizard** [reply]
Me too! Via blue tooth I think

**Reply by DuttyWizard** [reply]
Me too! Via blue tooth I think

**Comment by michael0643** [comment]
> Please I know nothing about ish or phishing or any of this, can someone please explain to me step by step how to make a Snapchat phishing page

Your a skid
